prime-dotnet-webapi/Models/EnrolleeLinkedEnrolments.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace Prime.Models
+{
+    [Table("EnrolleeLinkedEnrolments")]
+    public class EnrolleeLinkedEnrolments : BaseAuditable
```

Comment:
The class name should be singular, as well

Reply:
Fixed